### ISSUE TITLE
Galley imports not implemented

### ISSUE THREAD

**Issue by joemull** [BirkbeckCTP/imports#35]
I was under the impression that the Import / Export / Update tool could create galley files for imported XML or PDF files, but `utils.handle_file_import` doesn't recognize galleys or create Galley objects for them.

We should add logic to the mimetype recognition and set off the creation of the galley.

**Comment by joemull** [comment]
@mauromsl and @ajrbyers I thought of one more piece of logic--what if files that have a checksum that matches the most recent existing file are not imported? This would prevent large-scale duplication of identical content if multiple imports are run with the same data.

**Comment by mauromsl** [comment]
> @mauromsl and @ajrbyers I thought of one more piece of logic--what if files that have a checksum that matches the most recent existing file are not imported? This would prevent large-scale duplication of identical content if multiple imports are run with the same data.

yep, that's a great idea. It would save users from having to remove the files they have already imported if they wanted to re-run an impoort

**Comment by ajrbyers** [comment]
There is a func for calculating a files MD5 that we use already which could be useful @joemull

**Comment by joemull** [comment]
> 

excellent!

**Comment by joemull** [comment]
Just edited the bullet points above to reflect latest discussion of how we are planning to structure the file folders:

-- export.zip
    --> metadata.csv
    --> manuscripts
        --> 591.docx
        --> 592.docx
    --> galleys
        --> 592.pdf
        --> 592.xml
    -> images
        --> ??? (this may merit more discussion)
